src/library/scala/collection/Iterable.scala

```diff
@@ -259,7 +259,7 @@ trait IterableOps[+A, +CC[_], +C] extends Any with IterableOnce[A] with Iterable
     *        x >  0       if this.size >  otherSize
     *   }}}
     *  The method as implemented here does not call `size` directly; its running time
-    *  is `O(size min _size)` instead of `O(size)`. The method should be overwritten
+    *  is `O(size min _size)` instead of `O(size)`. The method should be overridden
```

Comment:
good catch!

(it used to be `_size` because I didn't want a name clash with `def size`, and I guess that got missed when renaming it to `otherSize`)